lib/helpers/utils.js

```diff
@@ -614,12 +614,19 @@ export function isPackageManagerAllowed(name, conflictingManagers, options) {
 // HTTP cache
 const gotHttpCache = new Map();
 
+function isCacheDisabled() {
+  const noCache = process.env.CDXGEN_NO_CACHE;
```

Comment:
We use the below style of checking in the codebase. Shall we make this consistent?

```["true", "1"].includes(process.env.CDXGEN_NO_CACHE)```

Reply:
Updated code to the suggested logic